vip_hci/conf/utils_conf.py

```diff
@@ -72,7 +72,7 @@ def save(self, filename):
     def load(cls, filename):
         try:
             data = np.load(filename)
-        except FileNotFoundError:
+        except:
```

Comment:
`np.load` also raises `EOFError: Ran out of input` and `OSError: Failed to interpret file '...' as a pickle`.